src/data/migrations/20180930225403-create-fog.js

```diff
@@ -268,6 +268,21 @@ module.exports = {
         field: 'fog_type_id',
         references: { model: 'FogTypes', key: 'id' },
         onDelete: 'set null'
+      },
```

Comment:
If you need to modify a model, please create a new sequelize migration.
Otherwise, someone that has a previous DB that has already run the migration will not get your changes.